test/test.js

```diff
@@ -60,7 +60,8 @@ describe('mercurius', function() {
       })
       .expect(function(res) {
         assert.equal(res.status, 200);
-        assert.equal(res.text, tokenToUnregister);
+        assert.equal(res.body.token, tokenToUnregister);
+        assert(res.body.machines);
```

Comment:
I'm now wondering if it shouldn't actually be an array - I mean `machines` bit...
Will we need any direct access to this in the front-end?
All functions are gonna be created for each machine in a loop...